cypress/e2e/chinese/landing/landing.cy.js

```diff
@@ -1,5 +1,7 @@
 describe('Landing', () => {
   before(() => {
+    // Update baseUrl to include current language
+    Cypress.config('baseUrl', 'http://localhost:8080/chinese/news');
```

Comment:
I assume to fix the tests, this also needs to be done in `ads.cy.js` for `/chinese`. However, I do not have enough context to be sure whether it makes sense considering ads in China are treated differently?

Reply:
We are not treating Chinese to be used exclusively in mainland China anymore. That is the aim of this migration is to cater to the Global Chinese-speaking audience.